infrastructure/zk/src/docker.ts

```diff
@@ -77,7 +81,9 @@ export async function pull() {
 
 export const command = new Command('docker').description('docker management');
 
-command.command('build <image>').description('build docker image').action(build);
-command.command('push <image>').description('build and push docker image').action(push);
+command.command('build <image> <tag>').description('build docker image').action(build);
+command.command('push <image> <tag>').description('build and push docker image').action(push);
```

Comment:
Do we need to add the tag in those commands? I guess we could just remove the tag from them and use latest, otherwise I don't see any difference between `buildFromTag` and `build` so as between `push` and `pushFromTag`

Reply:
I see now, could we specify this in the argument name? just to clarify that those tags are different